lib/contents_curl.php

```diff
@@ -0,0 +1,79 @@
+<?php
+function curlgetContents( $url, $params, $post=false){
+	$ch = curl_init();
+	curl_setopt($ch, CURLOPT_URL, $post ? $url : $url.'?'.http_build_query($params) );
+	curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
+
+	#curl_setopt($ch, CURLOPT_COOKIEJAR, '/tmp/rssbridge-fb-cookies.txt');
+	#curl_setopt($ch, CURLOPT_COOKIEFILE, '/tmp/rssbridge-fb-cookies.txt');
+	$files = array_diff(scandir(__DIR__.'/../proxylist/'), array('.', '..'));
+	$proxies = [];
+	foreach($files as $file) {
+		$proxies_str = file_get_contents(__DIR__.'/../proxylist/'.$file);
+		$proxies = array_merge($proxies, explode("\n", $proxies_str, -1));
+	}
+	$proxy = $proxies[array_rand($proxies)];
```

Comment:
Will this work if no proxy is specified (no files in the 'proxylist/' directory)?